rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -242,11 +252,57 @@ class PlayerImpl
     std::shared_ptr<rclcpp::GenericPublisher> publisher_;
     std::function<void(const rclcpp::SerializedMessage &)> publish_func_;
   };
+
+  class PlayerClient final
+  {
+public:
+    explicit PlayerClient(
+      std::shared_ptr<rclcpp::GenericClient> cli,
+      std::string service_name,
+      std::string service_event_type,
```

Comment:
Comments from MichaelOrlov

```suggestion
      const std::string & service_event_type,
```

Reply:
Updated.